### PR TITLE
chore(actions): rename calculated events to actions, remove flag

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

PRs over issues.

We renamed "actions" to "calculated events", but the feedback has been negative, and everyone's [indecisive](https://github.com/PostHog/product-internal/issues/344).

## Changes

Renames "calculated events" back to "actions".

Retains the new "data management" page and layout for events, uses the old table for actions.

Removes the "simplify actions" flag.

## Out of Scope

Outside the scope of this PR are all the bugs that currently exist with action edits. For example, when testing locally, I couldn't save tags. It's not clear when you need to hit the save button (changing filters) and when you don't (textfields). There are many easy wins to gain on these pages, but let's do this after.

## How did you test this code?

Tested the interface locally, relying on CI to catch any test regressions.

### CODE REVIEW

File: posthog/api/utils.py [hunk 1]
```diff
@@ -361,84 +361,31 @@ def select_event_definitions(fields: str):
...
-                {shared_conditions}
-                {ordering}
-            """
-            if is_enterprise
-            else f"""
-                {select_event_definitions(raw_event_definition_fields)}
-                {shared_conditions}
-                {ordering}
-            """
-        )
```
Comment (mariusandra) [inline posthog/api/utils.py:393]: I only kept this `if` condition and got rid of the rest, which used magical SQL ✨  to get actions and event definitions in one combined query.

**Comment by joethreepwood** [comment]
If we're reverting this for consistency then it's probably worth @pjhul and/or I doing a pass on site content and docs to bring them into alignment with this too.

**Review by joethreepwood — APPROVED** [review]
Not a code review, but a firm nod of approval.

**Review by annikaschmid — APPROVED** [review]
Looks good!

**Review by benjackwhite — APPROVED** [review]
Didn't notice anything wrong. Checked out locally and it all seemed to work 👍

**Review by Twixes — APPROVED** [review]
LGTM

**Pull request merged**